src/main/resources/db/changelog/changes/v1.1.0/insert-instances-definition.xml

```diff
@@ -103,8 +103,94 @@
               "filterValueGetter": "\"left\"(lower((jsonb -> 'metadata'::text) ->> 'updatedDate'::text), 600)",
               "valueFunction": "\"left\"(lower(:value), 600)",
               "visibleByDefault": true
+            },
```

Comment:
I know it's a bit of a pain, but is there any way this could be reformatted so that nested blocks are indented more? It helps a TON with readability